app/src/main/java/com/severianfw/picto/view/home/HomeFragment.kt

```diff
@@ -0,0 +1,28 @@
+package com.severianfw.picto.view.home
+
+import android.os.Bundle
+import androidx.fragment.app.Fragment
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import com.severianfw.picto.R
+import com.severianfw.picto.databinding.FragmentHomeBinding
+
+class HomeFragment : Fragment() {
+
+    private lateinit var binding: FragmentHomeBinding
```

Comment:
Make it this way to avoid memory leak
<img width="864" alt="image" src="https://user-images.githubusercontent.com/42958956/156159124-cd8b6752-d488-4f3a-a638-9b1d0d78ab2c.png">

Reference: https://developer.android.com/topic/libraries/view-binding#fragments